docs/docs/deploying-to-zeit-now.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Deploying to Now
+title: Deploying to ZEIT Now
```

Comment:
We need to add a redirect if we're changing the URL

Reply:
Done!